app/Services/LoadBalancingService.php

```
@@ -24,14 +25,15 @@ public function setServerPool(ServerPool $serverPool)
      */
     public function getLowestUsageServer(): ?Server
     {
-        return $this->servers
+        return $this->serverPool->servers()
```

PHPStan (2.2.7)

[error] Method App\Services\LoadBalancingService::getLowestUsageServer() should return App\Models\Server|null but returns stdClass|null.


(return.type)